# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1392735592 18000
#      Tue Feb 18 09:59:52 2014 -0500
# Branch stable
# Node ID 4c2465444a96683fd7bf3e612d2b25b61332cb04
# Parent  55e376fd8636b671a400deee9bab3463c7c90d0a
Use %!testif HAVE_UMFPACK for sparse lu test added in cset 2a45b6b87bee

* lu.cc: Use %!testif HAVE_UMFPACK for lu %!test on sparse data that was
added in cset 2a45b6b87bee.

diff --git a/libinterp/corefcn/lu.cc b/libinterp/corefcn/lu.cc
--- a/libinterp/corefcn/lu.cc
+++ b/libinterp/corefcn/lu.cc
@@ -569,17 +569,17 @@ information.\n\
 %! [l u p] = lu (single ([1, 2; 3, 4; 5, 6]));
 %! assert (l, single ([1, 0; 1/5, 1; 3/5, 1/2]), sqrt (eps ("single")));
 %! assert (u, single ([5, 6; 0, 4/5]), sqrt (eps ("single")));
 %! assert (p(:,:), single ([0, 0, 1; 1, 0, 0; 0 1 0]), sqrt (eps ("single")));
 
 %!error lu ()
 %!error <can not define pivoting threshold> lu ([1, 2; 3, 4], 2)
 
-%!test
+%!testif HAVE_UMFPACK
 %! Bi = [1 2 3 4 5 2 3 6 7 8 4 5 7 8 9];
 %! Bj = [1 3 4 5 6 7 8 9 11 12 13 14 15 16 17];
 %! Bv = [1 1 1 1 1 1 -1 1 1 1 1 -1 1 -1 1];
 %! B = sparse (Bi, Bj, Bv);
 %! [L, U] = lu (B);
 %! assert (L*U, B);
 %! [L, U, P] = lu(B);
 %! assert (P'*L*U, B);
